### ISSUE TITLE
Callback code

### ISSUE THREAD

**Issue by ghost** [jxnblk/SoundRad#23]
Callback code does not work.

When I connect to GitHub and verify my account, the callback is stuck and returning null.

**Comment by jxnblk** [comment]
Do you mean when you connect to SoundCloud?

**Comment by ghost** [comment]
Yep

**Comment by jxnblk** [comment]
What device and browser are you using?

**Comment by ghost** [comment]
2012 MacBook Pro, Safari 6.1/7.0 ML beta.

**Comment by jxnblk** [comment]
Is that Mavericks Beta?

**Comment by ghost** [comment]
Nope, just the browser. The Safari Beta is available for Mountain Lion, which is what I'm running.

**Comment by jxnblk** [comment]
Oh cool. I'll try downloading later and troubleshooting. In the meantime you can try the 'Connect with iOS7' link at the bottom of the home page.

**Comment by delroh** [comment]
Hi, I think I'm experiencing this issue too, but on Android 2.3 Gingerbread. Tried using the default browser and Dolphin browser mini.

Using Android's default browser, after I login to soundcloud, I still see the connect to soundcloud button and it doesn't matter if I refresh or touch the button again, it won't connect. 

Using Dolphin browser mini browser, I can't get past the "Connecting..." screen. It says "On iOS7, switch to the tab with SoundRad open" so I tried that (just being curious) and the soundrad tab doesn't change either.

Is there something else I could try to login or help solve the issue?
